src/CMakeLists.txt

```diff
@@ -194,6 +194,19 @@ if(CRC_OPT)
   )
 endif(CRC_OPT)
 
+if(CRC_ARMV8)
```

Comment:
What if user will use both CRC_OPT and CRC_ARMV8? Need some protection from this situation.